scripts/includes/common.sh

```diff
@@ -201,11 +199,9 @@ EOT"
     sudo mkdir -p ${CURRENT_USER_CLI_CONFIG_DIRECTORY}
   fi
 
-  sudo sh -c "cat <<EOT > ${CURRENT_USER_CLI_CONFIG_PATH}
-[credentials]
-username = ${USERNAME}
-password = ${PASSWORD}
-EOT"
+  sudo sh -c st2 login --config ${CURRENT_USER_CLI_CONFIG_PATH} \
```

Comment:
Any reason why using `sudo` and `sh` is required? Ideally is to avoid something like that, making sure that `st2` does the right thing.

Just `st2 login` would be enough.

Reply:
Alright, as described in another comment, 1 run `sudo st2 login -w` is needed to write config for root and another `st2 login -w` is needed to write user's config.